Bundle/CoreBundle/Resources/public/js/edit/widget.js

```diff
@@ -80,9 +81,10 @@ $vic(document).on('click', '.v-modal--widget a[data-modal="update"], .v-modal--w
             var formData = new FormData($vic(form)[0]);
             var contentType = false;
         }
+        var url = $vic(form).attr('action') + '?containerIds=' + containerIds.toString();
```

Comment:
I don't like this, I'd prefer to do this sooner, in php. You need to put it in the `$actionParams` variable and let the route generator do its job : https://github.com/jdubuisson/victoire/blob/cc2dc679c6ed033d1317a04425bfcebbe56fc067/Bundle/WidgetBundle/Builder/WidgetFormBuilder.php#L238